Lab 1 Changes (#251)
aws-samples/aws-hpc-tutorials: content/03-hpc-aws-parallelcluster-workshop/09-connect-cluster.md

tags = ["tutorial", "create", "ParallelCluster"]
+++

> Once we're connected to Pcluster Manager we can see the cluster we just created, depending on timing this cluster might be in **CREATE_IN_PROGRESS** or **CREATE_COMPLETE** state.

Reviewer: ```suggestion
Once you are connected to Pcluster Manager, you can see the created cluster, depending on timing this cluster might be in **CREATE_IN_PROGRESS** or **CREATE_COMPLETE** state.
```
